.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
+    -   repo: https://github.com/PyCQA/flake8
+        rev: 4.0.1
+        hooks:
+        -   id: flake8
+            args: [
```

Comment:
can you use the existing .flake8 file instead of duplicating args here?  otherwise need to always remember to update in two locations.

Reply:
Readme on how to install... or auto install it:
https://pre-commit.com/#automatically-enabling-pre-commit-on-repositories